### PR TITLE
Make requested revisions to recent PRs

### PR DESCRIPTION
**Pull request by Rangi42**
@ISSOtm Collecting your feedback on already-merged PRs here.

- https://github.com/gbdev/rgbds/commit/79948be3a0e09bd1acbac40e8f5fe0b80e6b6c32
- https://github.com/gbdev/rgbds/commit/ab42eb2c503a37e524290f9d84b55d1cc56f3bd0
- https://github.com/gbdev/rgbds/commit/80bdc889f86796307bf61556107505ae819e7775

### CODE REVIEW

File: man/rgbasm.5 [hunk 1]
```diff
@@ -414,8 +414,12 @@ Most of them return a string, however some of these functions actually return an
 .It Fn STRIN str1 str2 Ta Returns the first position of Ar str2 No in Ar str1 No or zero if it's not present Pq first character is position 1 .
 .It Fn STRRIN str1 str2 Ta Returns the last position of Ar str2 No in Ar str1 No or zero if it's not present Pq first character is position 1 .
 .It Fn STRSUB str pos len Ta Returns a substring from Ar str No starting at Ar pos No (first character is position 1, last is position -1) and Ar len No characters long. If Ar len No is not specified the substring continues to the end of Ar str .
-.It Fn STRUPR str Ta Returns Ar str No with all ASCII letters (`a-z`) in uppercase.
-.It Fn STRLWR str Ta Returns Ar str No with all ASCII letters (`A-Z`) in lowercase.
+.It Fn STRUPR str Ta Returns Ar str No with all ASCII letters
```
Comment (ISSOtm) [inline man/rgbasm.5:417]: TODO: check if this renders properly

File: man/rgbasm.1 [hunk 2]
```diff
@@ -167,7 +167,7 @@ This acts as if a
 was read before the input
 .Ar asmfile .
 .It Fl p Ar pad_value , Fl Fl pad-value Ar pad_value
-Use this as the fill value for
+Use this as the value for
```
Comment (ISSOtm) [inline man/rgbasm.1:170]: See this thread: https://github.com/gbdev/rgbds/commit/80bdc889f86796307bf61556107505ae819e7775#r131394569

**Review by ISSOtm — DISMISSED** [review]

File: man/rgbasm.5 (hunk 1 above)
Comment (Rangi42) [reply man/rgbasm.5:417]: Yes, it does; `man man/rgbasm.5`:
image

File: man/rgbasm.1 (hunk 2 above)
Comment (Rangi42) [reply man/rgbasm.1:170]: Copying it to here for future:

> > > > "Use this as the default value"?
> > > 
> > > We already say "The default is 0x00" below. I think having a default value for a default value is a bit confusing.
> > > 
> > > How about just "Use this as the value"?
> > 
> > "...directives in ROM sections, unless overridden.", maybe?
> 
> I copied the phrasing for `-Q`: "Use this as the precision of fixed-point numbers after the decimal point, unless they specify their own precision."

But yeah, I think the current phrasing is OK.

Comment (ISSOtm) [reply man/rgbasm.1:170]: Idk, it's the "own value(s)" phrasing that I feel doesn't jive well with `ds`. Perhaps passive voice would work better? "...unless values are explicitly specified"?

Comment (ISSOtm) [reply man/rgbasm.1:170]: Scratch that: I couldn't see the "overridden" change outside of the Files view; this sounds perfect!

**Review by ISSOtm — APPROVED** [review]
I looked at the other commits around that time, and there's nothing to add. Thank you for taking the time to make this!

**Pull request merged**